Cmb2arr.py

```diff
@@ -0,0 +1,11 @@
+import numpy as np
+
+if __name__ == '__main__':
+    first_array_1d = np.array([10, 20, 30])
+
+    second_array_1d = np.array([[40, 50, 60], [70, 80, 90]])
+    print(first_array_1d)
+    print(second_array_1d)
+
```

Comment:
This is a good solution, however there's a built-in numpy action with a similar functionality for your convenience.
Read more about numpy.nditer here-
https://numpy.org/doc/stable/reference/generated/numpy.nditer.html